tox.ini

```diff
@@ -8,18 +8,16 @@ setenv =
 passenv = HOME
 
 [testenv:lint]
+deps = ruff
 commands =
-     flake8 pytest_operator tests/unit tests/integration
-     black --check pytest_operator tests/unit tests/integration
-deps =
-     flake8
-     black
+    ruff check pytest_operator tests/unit tests/integration
+    ruff format --check pytest_operator tests/unit tests/integration
 
 [testenv:reformat]
+deps = ruff
```

Comment:
I would recommend pinning to a specific version.

otherwise things suddenly get reformatted in a random pr just because a new versions of ruff was released.